f11robo_bridge_node:
  ros__parameters:
    f11robo_bridge:
      topic_name:
        cmd_vel: "/cmd_vel"
      frame_id:
        odom: "odom"
        base_link: "base_link"
        imu_link: "imu_link"
      serial:
        port: "/dev/F11Robo"
        baudrate: 115200
      broadcast_period: 0.001
      max:
        vel: 0.3 #[m/s]
        angular: 0.5 #[rad/s]
